services/autoscaling/src/simcore_service_autoscaling/_meta.py

```diff
@@ -47,6 +48,26 @@ def get_summary() -> str:
 )
 
 
+APP_STARTED_DYNAMIC_BANNER_MSG = r"""
+      _                                  _
+     | |                                (_)
+   __| | _   _  _ __    __ _  _ __ ___   _   ___
+  / _` || | | || '_ \  / _` || '_ ` _ \ | | / __|
+ | (_| || |_| || | | || (_| || | | | | || || (__
+  \__,_| \__, ||_| |_| \__,_||_| |_| |_||_| \___|
+          __/ |
+         |___/
+"""
+
+APP_STARTED_DISABLED_BANNER_MSG = r"""
+      _  _              _      _            _
+     | |(_)            | |    | |          | |
+   __| | _  ___   __ _ | |__  | |  ___   __| |
+  / _` || |/ __| / _` || '_ \ | | / _ \ / _` |
+ | (_| || |\__ \| (_| || |_) || ||  __/| (_| |
+  \__,_||_||___/ \__,_||_.__/ |_| \___| \__,_|
+"""
+
 APP_FINISHED_BANNER_MSG = "{:=^100}".format(
```

Comment:
THOUGHT: Consider adapting utils in `servicelib.utils_meta`  (e.g. add possibility of banner in the message) to unify all this part of a service

Reply:
fixed